### PR TITLE
Changes Flasks a little

### PR DESCRIPTION
**Pull request by TrustyGun**
The captain now spawns with a golden flask, and the detective gets theirs back as well. Bartenders can also now vend regular flasks from the Booze-O-Mat!

The original flask's sprite has been to change to remove the ss13 engravement from it, now that it is not the captain's anymore. The new gold one keeps the logo.

:cl:
rscadd: After carefully investing in the market, the Captain had enough money to replace his silver flask with a golden one!
rscadd: The Detective is now bringing his personalized flask to work, due to the grim work he has to do.
rscadd: The Booze-O-Mat now stocks 3 flasks.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
I really liked the old captain's one though

**Comment by GunHog** [comment]
Pics?

**Comment by coiax** [comment]
I was thinking, what if using a welding tool on a metal flask raised the temperature of the chemical contents? Ghetto chemical heating, essentially.

**Comment by GunHog** [comment]
That is a good idea, which is consistent with beakers.

**Comment by coiax** [comment]
You can heat chemicals in beakers with a welding torch already?

**Comment by Supermichael777** [comment]
its how you do ghetto chem with no power. unfortunately the fact that their is a severe lack of ingredients in maint dampens it a bit. its part of goof chem because back on goon chems came in mixed containers in several spots so you could whip up drugs.

**Comment by coiax** [comment]
Anyway, more drinking storage for people to drink strong whiskey while narrating about the mean streets of Fatboy Camp Alpha, is a good idea. :+1:

**Comment by TrustyGun** [comment]
New Old!Flask sprite on the left, Gold!Flask on the right:

untitled

Also, I need to replace the flask bottles already in the captains' rooms on several maps, but the new map merger is giving me issues. Any help?

**Comment by coiax** [comment]
@TrustyGun hop on IRC, I'll see what I can do

**Comment by Zilenan91** [comment]
What do the bar flasks look like?

**Comment by TrustyGun** [comment]
Same as the sprite on the left

**Comment by Supermichael777** [comment]
you could have just made it flask and silver flask instead of map edits. also UGLY BLACK OUTLINE REEEEE

**Comment by Zilenan91** [comment]
Bar flasks though

:+1:  from me

**Comment by TrustyGun** [comment]
The one on the left is literally the same exact sprite as before, just without the engravement though.

**Comment by TrustyGun** [comment]
untitled

okay fine, I submit

File: code/game/objects/items/weapons/vending_items.dm [hunk 1]
```diff
@@ -34,8 +34,8 @@
 /obj/item/weapon/vending_refill/boozeomat
 	machine_name = "Booze-O-Mat"
 	icon_state = "refill_booze"
-	charges = list(52, 4, 0)//of 156 standard, 12 contraband
-	init_charges = list(52, 4, 0)
+	charges = list(55, 4, 0)//of 159 standard, 12 contraband
+	init_charges = list(55, 4, 0)
```
Comment (phil235) [inline code/game/objects/items/weapons/vending_items.dm]: don't you mean 53 ?

Comment (TrustyGun) [inline code/game/objects/items/weapons/vending_items.dm]: oh whoops

**Comment by Zilenan91** [comment]
wait shit flasks can hold 100 units right

**Comment by TrustyGun** [comment]
it's 60, 10 more then regular drinking glasses.

**Pull request merged**